tests/crypto/test_vectors.h

```diff
@@ -0,0 +1,102 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef __TEST_VECTORS_H__
+#define __TEST_VECTORS_H__
+
+#define TEST_VECTORS_COUNT 12
```

Comment:
Because `test_vectors` are static, this is known at compile time.
```suggestion
#define TEST_VECTORS_COUNT (sizeof(test_vectors) / sizeof(test_vectors_t))
```

Reply:
Thanks